### DISCUSSION TITLE
Welcome to PublicRelease_2020 Discussions!

### DISCUSSION THREAD

**Discussion by FLOSIC** [FLOSIC/PublicRelease_2020#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by suryaphy520** [comment]
1) While optimizing fod of CO2 i am facing error written as LOWSIC: LOWDEN EIGENVALUES TOO SMALL. How to resolve this ?

**Reply by jacks1ka** [reply]
This indicates that the Fermi orbitals being generated from the current set of FODs do not span the occupied space of the Kohn-Sham orbitals.  This can happen if two FODs are at the same position, or symmetry equivalent positions, or very near each other.  It can also happen if the FODs are not compatible with the electronic structure of the system, for example, if you arrange FODs expecting a triple bond, when the system has a double bond.  It can be difficult to see what FODs are misplaced. For first row atoms like C and O, a good rule of thumb is that optimal FODs will likely have roughly tetrahedral arrangements around the atoms, consistent with the "rule of 8".    You might try this as a starting set:
  11  0
  0.000  0.000  0.000
  0.000  0.000  2.215
  0.000  0.000 -2.215
  0.000  0.800  1.108
  0.000 -0.800  1.108
  0.800  0.000  3.015
 -0.800  0.000  3.015
  0.800  0.000 -1.108
 -0.800  0.000 -1.108
  0.000  0.800 -3.015
  0.000 -0.800 -3.015

**Reply by yohyamamoto** [reply]
Also, take a look at these CO2 FODs. 
FRMIDT:
    11    0       
        -0.0300103560        0.0000041013       -0.0000005895    
         0.0002171964        0.0000001834        2.2288269448    
         0.0002172292        0.0000000930       -2.2288277537    
        -0.8508154878        1.4743106910        2.3223613707    
        -0.8508216654       -1.4743097207        2.3223597592    
         1.6998563571       -0.0000040547        2.3232436031    
         0.0048309012        0.0000005213        1.2367789299    
        -0.8508146361        1.4743108589       -2.3223612183    
        -0.8508208027       -1.4743099009       -2.3223596425    
         1.6998567314       -0.0000040798       -2.3232429370    
         0.0048302448        0.0000005161       -1.2367770467    

geometry:
ALL-CAR001 =        0.000000        0.000000        0.000000 UPO
ALL-OXY001 =        0.000000        0.000000        2.215000 UPO
ALL-OXY002 =        0.000000        0.000000       -2.215000 UPO

**Comment by suryaphy520** [comment]
For All Electron calculation of transition metal we are facing problem in Fe, Co, Mn .
i)	For Fe the energy after 1st calculation becomes zero. And when we try 2nd calculation it is ended by showing 
         (  DIAGGE: failed in ewevge, error code 16
Matrix size: 1000  NDIM:    15
Aborting MPI ………
Abort (4096) on node 0 (rank 0 in comm 0); application called mpi_Abort (MPI_COMM_WORLD,4096) – process 0). i changed the fod by 0.02 bohr but the result is same. How it will be resolved ?

**Reply by jacks1ka** [reply]
These symptoms suggest an error in input somewhere.  This may be in SYMBOL or ISYMGEN, or in FRMORB.  If the latter, you may wish to view the supplementary material for this paper: https://doi.org/10.1063/1.4996498 which provides optimized FOD positions for all atoms in the first transition row.

**Reply by suryaphy520** [reply]
Thank you so much for your kind suggestion. But Is there any software/code to generate or visualize FODs for transition metals? We have tried FODMC, it did not work.

**Reply by Guo-user-123** [reply]
I also met the same problem about Aborting MPI ………, could you tell me how to solve the problem? when i set the MPI = N , the calculation is right .
